packages/extras/src/lib/components/Instancing/InstancedMeshes/InstancedMeshes.svelte

```diff
@@ -43,17 +42,19 @@
     )
   }
 
-  $: components = Array.isArray(meshes)
-    ? (getInstanceComponentsArray(meshes) as any)
-    : (getInstanceComponentsObject(meshes) as any)
+  let components = $derived(
+    Array.isArray(meshes)
+      ? (getInstanceComponentsArray(meshes) as any)
+      : (getInstanceComponentsObject(meshes) as any)
+  )
 
-  $: meshesArray = Array.isArray(meshes) ? meshes : Object.values(meshes)
-  $: filteredMeshesArray = meshesArray.filter((mesh) => mesh.isMesh)
+  let meshesArray = $derived(Array.isArray(meshes) ? meshes : Object.values(meshes))
+  let filteredMeshesArray = $derived(meshesArray.filter((mesh) => mesh.isMesh))
 </script>
 
 <InnerInstancedMeshes
   meshes={filteredMeshesArray}
-  {...$$restProps}
+  {...props}
 >
   <slot {components} />
```

Comment:
brain fart here on my part, but would this be our usual

```suggestion
  <slot ref={components} />
```
or doesn't make sense here, i dunno 😅